tests/10apidoc/30room-create.pl

```diff
@@ -312,8 +315,17 @@ sub matrix_create_room
       content => \%opts,
    )->then( sub {
       my ( $body ) = @_;
+      my $room_id = $body->{room_id};
 
-      Future->done( $body->{room_id}, $body->{room_alias} );
+      do_request_json_for( $user,
```

Comment:
I've seen in other places (for example, https://github.com/matrix-org/sytest/blob/97be5056b762a720b9adcb9c5866c6b9aa92ab40/tests/32room-versions.pl#L54) where the alias is generated via a sprintf instead of loading it from the server.

Should I use a sprintf here (and avoid a round trip to the server) as well?

Reply:
yes, I think that would be better. I don't think this function should be implicitly making extra requests to the server.

Reply:
Removed the extra request here: https://github.com/matrix-org/sytest/pull/1216/commits/5fe2531aa0366eefa149a4809b40e87034c519a4